### PR TITLE
[jwt-cpp] port update to add version 0.5.1

### PR DESCRIPTION
**Pull request by prince-chrismc**
**Describe the pull request**

Port update

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  I am still working on this PR

### CODE REVIEW

**Comment by prince-chrismc** [comment]
> This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!

This is the greatest thing ever! It makes it so much easier to contribute now 🎉

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
The 'paraview' breaks due CI basline error,just wait CI error fixed.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the version update!

**Comment by pyrotechnics-io** [comment]
This broke the build because of an out of date hash value - #19837

File: ports/jwt-cpp/portfile.cmake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Thalhammer/jwt-cpp
-    REF b45bc9994d8087c5ba7aa1f1588302f04ae84c83 # v0.5.0
+    REF 60122bbd8e6b9009c4633c1a33ea94c5b682034a # v0.5.1
     SHA512 d2c6e2174cd86c27b8866f6ea50a379a321788e91f97a0548e694d11469c05d93b0713e0c38b37a8f14a0aded4fc4635599dcfb50142e4496fb3c8b9f7e5f8cd
```
Comment (JackBoosY) [inline ports/jwt-cpp/portfile.cmake:5]: Our pipeline test didn't found the SHA512 is incorrecy, @BillyONeal do you know why?

Comment (BillyONeal) [reply ports/jwt-cpp/portfile.cmake:5]: The pipeline has artifact caching turned on which assumes that the SHA is correct. (The cache key is the SHA512)

**Pull request merged**